index.d.ts

```diff
@@ -17,4 +19,32 @@ export declare class DirectoryTree {
   extension?: string;
 }
 
+export class Stats {
```

Comment:
Let's include a comment with the link for the source of these. Easier for "future us".